packages/scss/src/components/inputs/file/_file.scss

```diff
@@ -1,30 +1,31 @@
 .file {
 	border: 2px dashed _theme("commons.divider.color");
-	border-radius: 3px;
+	border-radius: _theme("commons.border.radius");
 	display: block;
 	max-width: 30rem;
-	padding: .5rem _theme("spacings.standard") 1.5rem;
+	padding: _theme("spacings.biggest") _theme("spacings.standard");
 	position: relative;
 	text-align: center;
 }
 
 .file-icon {
-	font-size: 3rem;
-	color: _color("primary", "lighter");
+	font-size: 1.5rem;
+	color: _color("primary", "200");
+}
+
+.file-title {
+	margin-bottom: 0;
+	font-size: _theme("sizes.big.font-size");
+	font-weight: 600;
 }
 
 .file-or {
 	font-size: _theme("sizes.small.font-size");
 	margin-bottom: _theme("spacings.smallest");
 }
 
-.file-title {
-	margin: 0 0 .5rem;
-	font-size: _theme("sizes.big.font-size");
-}
-
 .file-progress {
-	margin: 1rem 0 .5rem;
+	margin: _theme("spacings.small") 0 _theme("spacings.smaller");
 }
 
 .file-input {
```

Comment:
Really bad for accessibility and prevents drag and drop too.
We should consider something like this I think: https://codepen.io/vincent-valentin/pen/KKVVomb

Reply:
What is your suggestion here ?

Reply:
Don't apply a `display: none` to the input but make it almost transparent and positioned over the area. (With a empty title for prevent any native tooltip.)

Reply:
Oh you're commenting something out of the PR that's why I couldn't understand. I agree with you, we should make these adjustments

Reply:
Yeah, I thought it was an opportunity. But don’t worry, we can do it in another pull request.